hspec-core/src/Test/Hspec/Core/Golden.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ExistentialQuantification #-}
```

Comment:
`ExistentialQuantification` are rarely needed.  There is often a better way to achieve things.